wetterdienst/provider/dwd/mosmix/api.py

```diff
@@ -497,19 +497,15 @@ def _all(self) -> pd.DataFrame:
 
         df = pd.read_fwf(
             StringIO(payload.read().decode(encoding="latin-1")),
-            skiprows=4,
+            skiprows=2,
```

Comment:
Because of `skiprows=4`, the first two stations have been skipped.
```
01001 ENJA JAN MAYEN             70.56   -8.40    10
01008 ENSB SVALBARD              78.15   15.28    29
```

But we can assume the file changed it format anyway, so it might have been correct before. By chance, does anyone have a previous version around?